### ISSUE TITLE
 sklearn.linear_model RidgeCV got an unexpected keyword argument 'normalize'

### ISSUE THREAD

**Issue by JBOE22175** [PacktPublishing/Modern-Time-Series-Forecasting-with-Python#25]
current installation from anaconda_env.yml installt scikit-learn 1.2.2.
argument normalize was remove in version 1.2.
Installing verion 1.1. shows only a deprecation warning but works.

**Comment by yashpackt** [comment]
Hey @JBOE22175 , thanks for sharing the update regarding the software update. Will share this bug with the author to inform them about the changes.
If there's anything else you would like to add, you can comment and let me know so I can try to resolve it at latest and if there's nothing more to add, I can close this issue. 
Thanks!

**Comment by manujosephv** [comment]
have pushed in a fix for this in #36
